exercise7/ListDirectoriesAndFiles11.java

```diff
@@ -0,0 +1,36 @@
+package exercise7;
+
+import java.io.File;
+
+public class ListDirectoriesAndFiles11 {
+	public static void main(String[] args) {
+		if(args.length==0) {
+			System.out.println("you have not entered anything...");
+			return;
+		}
+		else {
+			String directory = args[0];
+			File file = new File("/Users/mukundyadav/Documents/GitHub");
```

Comment:
Instead of using the hard-coded path, can you use the `directory` which you received as arguments